client/src/components/Navigation/index.js

```diff
@@ -6,7 +6,7 @@ const NavContainer = styled.section`
   flex-wrap: wrap; 
```

Comment:
Ryan I moved this code to Styles folder. It is Labled Main. Pull down changes to accept or not accept the. Otherwise this will break